app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command, current_file, new_file = command.split()
+        dir_name = new_file.split("/")
+        if len(dir_name) == 2:
+            os.mkdir("first_dir")
+        if len(dir_name) == 3:
+            os.mkdir("first_dir/second_dir")
+        if len(dir_name) == 4:
+            os.mkdir("first_dir/second_dir/third_dir")
```

Comment:
this is not true

why do you create first_dir, second_dir .....

you need to use the names that will be passed in the command